modules/dbnd/src/dbnd/_core/task_run/task_run_runner.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import signal
 import typing
+import webbrowser
```

Comment:
let's not add it standard requriements, but to additional only ( I would do just import, and if error - write a good message) 

we are trying to keep core library (dbnd) very lean

Reply:
I'm not sure what you mean - do you want me to import `webbrowser` only when its needed? Something like this?

```
            if run_config.open_web_tracker_in_browser:
                import webbrowser
                webbrowser.open_new_tab(task_run.task_tracker_url)
```

Reply:
sorry. my bad, I was sure it's third party library.

Reply:
no problem ^^